module-code/app/securesocial/controllers/Assets.scala

```diff
@@ -1,3 +1,10 @@
 package securesocial.controllers
 
-class Assets extends controllers.AssetsBuilder(play.api.http.LazyHttpErrorHandler)
\ No newline at end of file
+import javax.inject.{ Inject, Singleton }
+
+import controllers.AssetsMetadata
+import play.api.http.HttpErrorHandler
+
+@Singleton
```

Comment:
This is a question. Why is this `@Singleton` necessary?

Reply:
May be this singleton annotation is added for custom Guice rules. I will remove it.